quarkchain/cluster/cluster_config.py

```diff
@@ -167,6 +167,15 @@ class MonitoringConfig(BaseConfig):
     ERRORS = "error"
 
 
+class PrometheusConfig(BaseConfig):
+    TIMEOUT = 10
+    HOST = "http://localhost:38391"
+    GAP = 30  # Interval between two total_balance queries
+    TOKEN = 0x8BB0
+    PORT = 8000  # Prometheus client expose port
+    ROOTBLOCK_HEIGHT = 1
```

Comment:
we don't want to hard code this number, right? what's the plan for production use?